msm/src/circuit_design/witness.rs

```diff
@@ -28,10 +38,24 @@ pub struct WitnessBuilderEnv<F: PrimeField, const CIX_COL_N: usize, LT: LookupTa
     /// each row is a map from lookup type to a vector of concrete
     /// lookups requested.
     pub lookups: Vec<BTreeMap<LT, Vec<Logup<F, LT>>>>,
+
+    /// Fixed values for selector columns. `fixed_selectors[i][j]` is the
+    /// value for row #j of the selector #i.
+    pub fixed_selectors: Vec<Vec<F>>,
+
+    // Not strictly needed at this point, but likely will be used later.
+    pub phantom_cix: PhantomData<CIx>,
```

Comment:
Maybe explain what this could potentially be, doesn't bring as much meaning to the struct as the other names `lookups`, `fixed_selectors`, etc.

Reply:
Will add a comment, but generally `phantom` fields are needed when one wants to parameterise a struct with a type parameter that is not used otherwise. 

E.g. sometimes `struct EmptyFieldTag<T: Field>(tag_ix: usize)` won't compile, and then `struct EmptyFieldTag<T: Field>{ tag_ix: usize, phantom: PhantomData<T> }` does the trick. https://doc.rust-lang.org/std/marker/struct.PhantomData.html